CHANGELOG.md

```diff
@@ -2,11 +2,13 @@
 
 [Full changelog](https://github.com/mozilla/glean/compare/v34.1.0...main)
 
+* General
+  * Introduce a new API `Ping#test_before_next_submit` to run a callback right before a custom ping is submitted.
```

Comment:
Annotate with PR# and a list of languages you're supporting, if you please.

Reply:
Will do